packages/apps-v5/heroku.yml

```diff
@@ -0,0 +1,24 @@
+setup:
```

Comment:
In hindsight, I think stubbing/replacing some functionality might be better than having this at the root.
Maybe just add a great explanation in a comment in this file for why it exists?

Reply:
Yes, I'm very curious about this.

Reply:
yes, I'm very curious about this

Reply:
The yaml file ended up being at the root level when checking where the `readFile()` path was navigating. I agree with stubbing the readManifest function that returns the information read by `readFile()`. Ideally we wouldn't want to keep a yaml file in a root directory if we don't have to. I know we were initially apprehensive on using `proxyquire` to address this, but I can definitely update that.

Reply:
I think stubbing it to read from another path, or just returning an object you hard code would be better.